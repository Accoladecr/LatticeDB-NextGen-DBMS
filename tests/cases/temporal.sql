CREATE TABLE t (
                   id TEXT PRIMARY KEY,
                   v INT
);
INSERT INTO t (id, v) VALUES ('a', 1);
UPDATE t SET v=2 WHERE id='a';
SELECT id, v FROM t FOR SYSTEM_TIME AS OF TX 1;
EXIT;
