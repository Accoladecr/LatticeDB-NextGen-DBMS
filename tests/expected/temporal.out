CREATE TABLE
INSERT 1 row(s)
UPDATE 1 row(s)
id | v
'a' | 1
Bye.
